docs/uci_defaults.txt

```diff
@@ -0,0 +1,42 @@
+---
+---
+UCI defaults
+============
+
+== Integrating custom settings through UCI defaults
+
+LEDE allows you to add custom settings by adding scripts into the 'files/etc/uci-defaults/' directory, which
+will then be run after flashing. When keeping settings, this should mean the scripts are run *after* the upgrade
+process flashed the image and appended the saved settings to the JFFS partition (which will be mounted as 
+'/overlay'). The path is identical for the buildroot and the image generator. Scripts should not be executable.
+To ensure your scripts are not interfering with any other scripts, make sure they get executed last by giving them
+a high prefix (e.g. 99_customisations). A basic script could look like this:
```

Comment:
this isn't actually true.  I've raised it before iirc.  They're alphabetical, but package names are after numbers, so zzzzzzzzzzz_yourscriptname is more reliable than 99_yourscript.  You can do something like https://github.com/remakeelectric/openwrt-cc/commit/9f83c196763641f1e11e7f8cc3712e9adc43f3f8 to see the real order as they progress.